lib/console.js

```diff
@@ -166,6 +170,36 @@ Console.prototype.assert = function assert(expression, ...args) {
   }
 };
 
+// Defined by: https://console.spec.whatwg.org/#clear
+Console.prototype.clear = function clear() {
+  // It only makes sense to clear if _stdout is a TTY.
+  // Otherwise, do nothing.
+  if (this._stdout.isTTY) {
+    const { cursorTo, clearScreenDown } = require('readline');
+    cursorTo(this._stdout, 0, 0);
+    clearScreenDown(this._stdout);
+  }
+};
+
+// Defined by: https://console.spec.whatwg.org/#count
+Console.prototype.count = function count(label = kDefaultCount) {
```

Comment:
Yeah, I saw that but looking at the browser implementations, neither Chrome or Firefox treat `console.count()` and `console.count('default')` as being equivalent. They implement separate counters.